lib/airbrake.js

```diff
@@ -46,7 +47,7 @@ function Airbrake() {
   ];
 }
 
-_.assign(Airbrake.prototype, EventEmitter.prototype);
+assign(Airbrake.prototype, EventEmitter.prototype);
```

Comment:
Sure. I can make the change but I can't say for sure that it wouldn't break something.